src/api/instances.tsx

```diff
@@ -333,3 +337,30 @@ export const fetchInstanceLogFile = (
       .catch(reject);
   });
 };
+
+export const duplicateInstance = (
+  values: LxdInstanceDuplicate,
+  instance: LxdInstance,
+): Promise<LxdOperationResponse> => {
+  return new Promise((resolve, reject) => {
+    fetch(`/1.0/instances`, {
+      method: "POST",
+      body: JSON.stringify({
+        //Does not yet consider targetClusterMember or storagePool
+        description: instance.description,
+        name: values.instanceName,
+        source: {
+          allow_inconsistent: values.allowInconsistent,
+          instance_only: values.instanceOnly,
+          project: values.project,
+          source: instance.name,
+          type: "copy",
+        },
+        stateful: values.stateful,
```

Comment:
This transformation should not happen in the API layer, but in the form component. Here we can receive the json directly. Actually I think we do not need a new endpoint here at all, but can reuse the `createInstance` one above.